source/localizable/seminar/index.nl.html.erb

```diff
@@ -1 +1,247 @@
-Seminar here ...
```

Comment:
Ye it does, but moved the pages and its partials to sub directories to keep it clean.
`seminar/index.nl.html.erb` has the same result as `seminar.nl.html.erb`.